conductr_cli/terminal.py

```diff
@@ -42,6 +42,26 @@ def docker_machine_ip(vm_name):
     return subprocess.check_output(cmd, universal_newlines=True).strip()
 
 
+def docker_machine_create_vm(vm_name):
+    cmd = ['docker-machine', 'create', vm_name, '-d', 'virtualbox']
```

Comment:
Right but Vmware fusion could also be an option when running locally... Perhaps others?

> On 31 Dec 2015, at 2:21 PM, Felix Satyaputra <notifications@github.com> wrote:
> 
> In conductr_cli/terminal.py:
> 
> > @@ -42,6 +42,26 @@ def docker_machine_ip(vm_name):
> >      return subprocess.check_output(cmd, universal_newlines=True).strip()
> >  
> >  
> > +def docker_machine_create_vm(vm_name):
> > +    cmd = ['docker-machine', 'create', vm_name, '-d', 'virtualbox']
> I think so since it's expected that the sandbox will be running locally, and hence it will be on Virtualbox.
> 
> The other drivers are: amazonec2, azure, digitalocean, exoscale, generic, google, none, openstack, rackspace, softlayer, virtualbox, vmwarefusion, vmwarevcloudair, vmwarevsphere.
> 
> —
> Reply to this email directly or view it on GitHub.
>